### PR TITLE
Transition depreciated @red-hat-insights comopnent to @redhat-cloud-services, updates all non-breaking deps 🦹‍♀️🤗 

### PR DESCRIPTION
**Pull request by AllenBW**
fixes https://projects.engineering.redhat.com/browse/RHCLOUD-6509

#### still looks stunning
Screen Shot 2020-05-13 at 11 34 06 AM

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -1,37 +1,65 @@
...
+    &__accent {
+        color: var(--pf-global--palette--black-500);
+    }
 
-    &__emphasis { @include rem("font-size", 32px); }
+    &__emphasis {
+        @include rem("font-size", 32px);
+    }
 
     &__icon {
```
Comment (christiemolloy) [inline src/App.scss:21]: I might be wrong, but I don't know if we use all of these colors?

Comment (AllenBW) [reply src/App.scss:21]: Ooooo no idea, not keen on touching that, this is just auto linting cuz the file was touched

Comment (christiemolloy) [reply src/App.scss:21]: ahhh ok, all good then!

**Comment by AllenBW** [comment]
In we go! gonna hold off on stable push till everything is 🍍

**Pull request merged**